### PR TITLE
Responsive Quize Result Page

### PR DESCRIPTION
**Pull request by aijazraja**
# Description

responsive  Quiz Result Page. I have changed CSS by adding Media Query in Quiz Result page Container and component Card Result Page(Beth's Component).

Fixes # (issue)

# How to test?
Storybook
You have to change the file name by adding .component to file names of components I  am using here. It is because we merge the Quiz result page component last week with the .component extension in file names. But Anqira PR is still not merged.

# Checklist

- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by TITHI1244 — CHANGES_REQUESTED** [review]
Hello Aijaz. I think, we are supposed to add the page path also in the App router, something like this:
```
<Route exact path="/{path_name}">
      <{Your_component_name} />
 </Route>
```
 Then it would be easier to review :)

**Review by TITHI1244 — CHANGES_REQUESTED** [review]
@aijazraja , can you please change all the pixels into % or em? As I can see, all others are coding like that :)

**Review by dpfernandes — APPROVED** [review]
👍

**Review by qamarfj — APPROVED** [review]

**Review by cph-kiwi — APPROVED** [review]

File: src/client/components/CardProfileResult/CardProfileResultComponent.styles.css [hunk 1]
```diff
@@ -13,8 +12,8 @@
 
 .card-profile-result-list-items {
   background-color: #f0f0f0;
-  height: 420px;
-  width: 440px;
+  min-height: 420px;
+  max-width: 440px;
   padding-top: 30px;
```
Comment (bytewiz) [inline src/client/components/CardProfileResult/CardProfileResultComponent.styles.css:17]: Be careful with setting fixed sizes :)

**Review by bytewiz — APPROVED** [review]

**Review by TITHI1244 — APPROVED** [review]

**Pull request merged**